devel/icu-devel/Portfile

```diff
@@ -116,10 +116,20 @@ if {${subport} eq ${name} || ${subport} eq "${name}-lx"} {
     test.target     check
     test.args       VERBOSE=1
 
+    # icu4c requires python2.7+ regardless of configure:
+    #   checking for python3...
+    # libxml which is used to compile llvm/clang depends on icu,
+    # and adding MacPorts' python may introduce a dependency loop.
```

Comment:
Do you really think it will cause a dependency loop? The port has been depending on python27 already for a long time without (as far as I know) any problems. The libxml2 and icu and python27 ports already include the clang_dependency portgroup, which instructs MacPorts to avoid using clang 5 or later, which should avoid a dependency cycle. It would be good not to make the user install yet another python port (python27-bootstrap) unless that's really needed.

I note however that icu-devel does not include the clang_dependency portgroup, and although icu-devel is not a clang dependency, icu is, and the purpose of icu-devel is to test the suitability of a new version of icu prior to updating the main port, so it might be a good idea to make icu-devel use the clang_dependency portgroup too so that it uses the same (older) compiler that icu eventually will, to ensure there are no problems with that.

Reply:
Right now python27 is enough to build it. As you read in the ticket some users are confused by this configure line and I'd like to keep it clean.

Anyway, in the future it might change.

Reply:
So again, why would switching from python27 to python27-bootstrap be good?

Reply:
@ryandesign to minimise dependency tree.

But, after some thinking I feel that I'm overengenering here. Let me revert this changes.